src/main/kotlin/crypto_service/controller/CryptoController.kt

```diff
@@ -33,35 +33,28 @@ class CryptoController(
     @GetMapping("/encrypt")
     fun encrypt(
         text: String,
-        config: String,
+        _config: String,
         gcpAccessToken: String,
         riScId: String
     ): ResponseEntity<String> {
-
-        // Hva skjer om decodingen feiler?
         val urlDecodedText = URLDecoder.decode(text, StandardCharsets.UTF_8.toString())
-
-        val encryptedString = encryptionService.encrypt(urlDecodedText, config, GCPAccessToken(gcpAccessToken), riScId)
+        val encryptedString = encryptionService.encrypt(urlDecodedText, _config, GCPAccessToken(gcpAccessToken), riScId)
 
         return ResponseEntity.ok().body(encryptedString)
     }
 
-//    data class EncryptionRequest(
-//        val text: String,
-//        val config: String,
-//        val gcpAccessToken: String,
-//        val riScId: String
-//    )
 
     @PostMapping("/encryptPost")
```

Comment:
Generelt spørsmål, trenger vi å kalle endepunktet for encrypt**Post**? Holder det at endepunktet heter `encrypt` og at det er av typen `Post`?